publish.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+set -xe
+
+# Install local teads-central
+docker run --rm -i docker-registry.teads.net/curl:master -sL http://ebzpub.ebuzzing.com/files/teads-central/get.sh | sh - 
+
+IMAGE=$(./teads-central vars image)
+HASH=$(./teads-central vars hash)
```

Comment:
that's only if the `test.sh` tags it with HASH - have a look at other small repos for the usual test.sh/publish.sh